src/main/archetype/ui.tests/test-module/rules/template-rules.json

```diff
@@ -0,0 +1,83 @@
+{
+  "template": {
```

Comment:
I don't understand the concept of these rules file ? How would one know more details on this, for example, if someone wants to change this, its very difficult.

Reply:
@rismehta, I have mentioned about this rule in coment https://github.com/adobe/aem-project-archetype/pull/720#discussion_r678838866. We can add this information to ReadMe as well. The rule shows forms hierarchical structure. Isolating the verification criteria in rule will help in keeping test case code short using some utility functions to verify according to rule file. In future if there are more validation points need to be required, adding them to the rule file will still keep the test code maintainable otherwise it might be a lengthy script.